package logica.excepciones;

public class UsuarioException extends Exception{
    public UsuarioException(){super("Datos inv√°lidos o ya existe dicho userName, probar nuevamente. Gracias");}
}
